doc/source/ray-air/examples/convert_existing_pytorch_code_to_ray_air.ipynb

```diff
@@ -576,62 +577,62 @@
     "\n",
     "1. We import Ray Train and Ray AIR Session:\n",
     "\n",
-    "```python\n",
-    "import ray.train as train\n",
-    "from ray.air import session\n",
-    "```\n",
+    "    ```python\n",
+    "    import ray.train as train\n",
+    "    from ray.air import session\n",
+    "    ```\n",
     "\n",
     "\n",
     "2. We use a `config` dict to configure some hyperparameters (this is not strictly needed but good practice, especially if you want to o hyperparameter tuning later):\n",
     "\n",
-    "```python\n",
-    "def train_func(config: dict):\n",
-    "    batch_size = config[\"batch_size\"]\n",
-    "    lr = config[\"lr\"]\n",
-    "    epochs = config[\"epochs\"]\n",
-    "```\n",
+    "    ```python\n",
+    "    def train_func(config: dict):\n",
+    "        batch_size = config[\"batch_size\"]\n",
+    "        lr = config[\"lr\"]\n",
+    "        epochs = config[\"epochs\"]\n",
+    "    ```\n",
     "\n",
     "3. We dynamically adjust the worker batch size according to the number of workers:\n",
     "\n",
-    "```python\n",
+    "    ```python\n",
     "    batch_size_per_worker = batch_size // session.get_world_size()\n",
```

Comment:
nit: I think these indentations were intentional because they are meant to show code inside the `train_func`, do you think it is preferable to unindent them?

Reply:
Oh I see. We can't unindent because the code must be in the same column or to the right of *```python*. Can we add another indent to retain the structure of `train_func` like this? It looks much better.

<img width="531" alt="Screenshot 2023-05-10 at 2 02 41 PM" src="https://github.com/ray-project/ray/assets/26745457/614229c4-7021-485b-a290-8c11350977e9">

Reply:
Oh yeah that's exactly what I meant i.e. indentations of the code relative to ```python!

Reply:
👍